xdebug.c

```diff
@@ -106,54 +106,185 @@ static SIZETorINT (*xdebug_orig_ub_write)(const char *string, SIZETorUINT len TS
 
 static int xdebug_trigger_enabled(int setting, char *var_name, char *var_value TSRMLS_DC);
 
+ZEND_BEGIN_ARG_INFO_EX(xdebug_get_stack_depth_args, ZEND_SEND_BY_VAL, ZEND_RETURN_VALUE, 0)
+ZEND_END_ARG_INFO()
```

Comment:
In the MongoDB driver, we created a special ARG INFO struct for functions/methods without arguments, and reused that for all these methods.

If we would to replicate that for Xdebug, there would be a ```xdebug_void_args``` that gets defined once, but used 20 or so times. It means less code to maintain.